ppr-ui/src/enums/statusTypes.ts

```diff
@@ -12,3 +12,10 @@ export enum UIStatusTypes {
   EXPIRED = 'Expired',
   DISCHARGED = 'Discharged'
 }
+
+export enum mhUIStatusTypes {
+  DRAFT = 'Draft',
+  ACTIVE = 'Active',
+  EXEMPT = 'Exempt',
+  HISTORICAL = 'Historical'
```

Comment:
Just curious, these status types i'm sure are valid types but when we sort by these types in the table the results of that filter just yield all the active registrations. Is this known or expected at this point?

Reply:
Good catch... I added mhAPIStatusTypes with values uppercased.  API expects Uppercase for status.